README.md

```diff
@@ -8,11 +8,12 @@ Private WireGuard® networks made easy
 
 This repository contains all the open source Tailscale client code and
 the `tailscaled` daemon and `tailscale` CLI tool. The `tailscaled`
-daemon runs primarily on Linux; it also works to varying degrees on
-FreeBSD, OpenBSD, Darwin, and Windows.
+daemon runs on Linux, Windows and macOS, and to varying degrees on FreeBSD, OpenBSD, and Darwin. (Note that Tailscale runs on iOS and Android as well, but not with `tailscaled`.)
```

Comment:
link the word "macOS" to https://tailscale.com/kb/1065/macos-variants/ ?

And for the parenthetical, perhaps: (The Tailscale iOS and Android apps use this repo's code, but this repo doesn't contain the mobile GUI code.)